docs/en_us/platform_api/source/profile_images/endpoints.rst

```diff
@@ -0,0 +1,19 @@
+################################################
+Profile Images API Endpoints
+################################################
+
+You use the User API to view information about users and update
```

Comment:
Should this say more about what the Profile Image API endpoints are for? It seems confusing that it just repeats what the User API is for.